benchmark-service/benchmark/templates/index.html

```diff
@@ -2,11 +2,15 @@
 {% block title %} Benchmarks {% endblock %}
 
 {% block content %}
+<link rel="stylesheet" href="{{ base_path }}/static/css/autocomplete.css" />
+
 <h1>Benchmark System</h1>
 
-<form action="{{ base_path }}/" method="get">
-    <input type="text" name="file"></input>
-    <input type="submit" value="lookup"></input>
+<form autocomplete="off" action="{{ base_path }}/" method="get">
+    <div class="autocomplete">
+        <input type="text" name="file" id="autoComplete"></input>
```

Comment:
I see the code below sets the default that's listed as gs://..., but your image is empty for the lookup without any text in it. Is there a way to get gs://... to be the placeholder?

Reply:
it is there but only when hovering or clicking on the box, I tried to get it to show up normally also but it's not working..

Reply:
working now